debian/couchdb-nouveau.conffiles

```diff
@@ -0,0 +1 @@
+/opt/nouveau/etc/nouveau.yaml
```

Comment:
Ah nice. I totally forgot we had two packages for debian (for RPM we have only one).  The packages do build now. 

This potentially then could be a package upgrade (3.4.3.1 for deb).

But we should check how upgrading works from 3.4.3 where this files goes from a non-conffile to a conffile. And then figure out if we could test the upgrade from 3.4.3.1 to something higher, like 3.4.3.2, with some changes to it to see that it refuses to overwrite it.

Reply:
Jonas confirmed this did the right thing in slack and it appears to for me in a local dev env. it's weird that debian docs only say `conffiles` and not `package.conffiles` even though other things follow that pattern. but this apparently works.

Reply:
> Jonas confirmed this did the right thing in slack and it appears to for me in a local dev env. 

Thanks

> debian docs only say conffiles and not package.conffiles even though other things follow that pattern. but this apparently works.

I think both work, some docs mention it as sub-packages are a bit of an obscure feature. I did find a reference to `$package.conffiles` in https://manpages.debian.org/bookworm/debhelper/dh_installdeb.1.en.html